Cargo.toml

```diff
@@ -90,6 +92,7 @@ features = [
 wayland-client = { version = "0.29", default_features = false,  features = ["use_system_lib"], optional = true }
 wayland-protocols = { version = "0.29", features = [ "staging_protocols"], optional = true }
 sctk = { package = "smithay-client-toolkit", version = "0.15.4", default_features = false, features = ["calloop"],  optional = true }
+sctk-adwaita = { version = "0.3", optional = true }
```

Comment:
Could you explicitly specify patch version you've used, since proper text rendering isn't on `0.3`, but on some release.

Reply:
Done